src/dlgTriggerEditor.cpp

```diff
@@ -1111,21 +1111,10 @@ void dlgTriggerEditor::slot_search_triggers( const QString s )
             {
                 QStringList sl;
                 sl << "Key" << key->getName() << "name";
-                if( ! parent )
```

Comment:
The problem with this specific change is that while the code is currently dead and removing it keeps the current behavior, I'm not sure that it's correct because I don't know what it's supposed to do.

For example, moving parent outside the for loop would make this code live again and there would be a shared parent throughout the loop. Given that I'm not sure how to test this, I'm not sure if that even makes sense.

If you want, you can try moving line 1108 to 1104 and see if the behavior 'improves'. It might crash or do something crazy or it might be awesome. It might not even change, though that's unlikely.